vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_info.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+
+#include <utility>
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(
+    PipelineEmbeddingInfo&& info) noexcept = default;
+
+PipelineEmbeddingInfo& PipelineEmbeddingInfo::operator=(
+    PipelineEmbeddingInfo&& info) noexcept = default;
+
+PipelineEmbeddingInfo::~PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(
+    const int version,
+    const std::string& timestamp,
+    const std::string& locale,
+    const int embeddings_dim,
```

Comment:
do we need `embeddings_` as this is part of PipelineEmbedding? Maybe just `dim`

Reply:
Updated to dim in all relevant places